From 4250949f13234323262771e9e2976019ecaea33d Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Sat, 28 Apr 2012 17:56:38 -0300
Subject: [PATCH 20/40] =?UTF-8?q?4.3.1=20Corona=20de=20Hielo:=20Pura=20Diver?=
 =?UTF-8?q?si=C3=B3n?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Ajustado SAI del NPC "El Ocular", para que ataque a los jugadores al ser golpeado con la spell
---
 .../2012_04_03_100_world_its_all_fun_and_games.sql |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_03_100_world_its_all_fun_and_games.sql

diff --git a/sql/updates/world/reanemu/2012_04_03_100_world_its_all_fun_and_games.sql b/sql/updates/world/reanemu/2012_04_03_100_world_its_all_fun_and_games.sql
new file mode 100644
index 0000000..5431fdb
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_03_100_world_its_all_fun_and_games.sql
@@ -0,0 +1,8 @@
+-- The Ocular SAI
+SET @EYE := 29747;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@EYE AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(@EYE,0,0,0,0,0,100,0,1500,1500,6000,6000,11,55269,0,0,0,0,0,2,0,0,0,0,0,0,0,'Ocular - cast Deathly Stare every 5 secs'),
+(@EYE,0,1,0,6,0,100,0,0,0,0,0,33,29803,0,0,0,0,0,7,0,0,0,0,0,0,0,'Ocular - cast It\'s All Fun and Games: The Ocular On Death'),
+(@EYE,0,2,0,8,0,100,1,30740,0,0,0,49,0,0,0,0,0,0,7,0,0,0,0,0,0,0,'Ocular - Attack start on spell hit'),
+(@EYE,0,3,0,25,0,100,0,0,0,0,0,75,55162,1,0,0,0,0,1,0,0,0,0,0,0,0,'Ocular - cast Transform on Spawn');
-- 
1.7.8.msysgit.0

